specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1463 @@
+          "type": "integer",
+          "example": 400052
+        },
+        "progress": {
+          "format": "float",
+          "description": "Progress of the translation if available",
+          "maximum": 1,
+          "minimum": 0,
+          "type": "number",
+          "example": 0.5
+        },
+        "id": {
+          "format": "int64",
+          "description": "Document Id",
+          "type": "integer"
+        }
+      }
+    },
+    "DocumentStatusResponse": {
+      "description": "Document Status Response",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The detail status of individual documents",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DocumentStatusDetail"
+          }
+        },
+        "@nextLink": {
+          "description": "Url for the next page.  Null if no more pages available",
+          "type": "string",
+          "example": "https://westus.cognitiveservices.azure.com/translator/text/batch/v1.0.preview.1/operations/727BF148-F327-47A0-9481-ABAE6362F11E/documents?$top=5&$skip=15"
+        }
+      }
+    },
+    "StatusSummary": {
+      "type": "object",
+      "properties": {
+        "total": {
+          "format": "int32",
+          "description": "Total count",
+          "type": "integer",
+          "example": 1
+        },
+        "failed": {
+          "format": "int32",
+          "description": "Failed count",
+          "type": "integer",
+          "example": 0
+        },
+        "success": {
+          "format": "int32",
+          "description": "Number of Success",
+          "type": "integer",
+          "example": 1
+        },
+        "inProgress": {
+          "format": "int32",
+          "description": "Number of in progress",
+          "type": "integer",
+          "example": 0
+        },
+        "notYetStarted": {
+          "format": "int32",
+          "description": "Count of not yet started",
+          "type": "integer",
+          "example": 0
+        }
+      }
+    },
+    "OperationStatusSummary": {
+      "description": "Job status response",
+      "required": [
+        "createdDateTimeUtc",
+        "id",
+        "lastActionDateTimeUtc",
+        "summary"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "description": "Id of the operation.",
+          "type": "string",
+          "example": "36724748-f7a0-4db7-b7fd-f041ddc75033"
+        },
+        "createdDateTimeUtc": {
+          "format": "date-time",
+          "description": "Operation created date time",
+          "type": "string",
+          "example": "2020-01-20T11:40:07.7010000-08:00"
+        },
+        "lastActionDateTimeUtc": {
+          "format": "date-time",
+          "description": "Date time in which the operation's status has been updated",
+          "type": "string",
+          "example": "2020-01-20T11:41:07.7010000-08:00"
```

Comment:
Please don't go beyond millisecond precision in date-time values. It will cause problems when consuming your service from various languages (e.g. javascript)